strawberry/schema/schema_converter.py

```diff
@@ -164,7 +166,11 @@ def from_field(self, field: StrawberryField) -> GraphQLField:
         graphql_arguments = {}
         for argument in field.arguments:
             assert argument.name is not None
-            graphql_arguments[argument.name] = self.from_argument(argument)
+            argument_name = (
+                to_camel_case(argument.name) if self.auto_camel_case else argument.name
+            )
+
+            graphql_arguments[argument_name] = self.from_argument(argument)
```

Comment:
this should become a method: `self.get_graphql_name(argument.name)` or even `self.get_graphql_name(argument)` when we unify things :)